src/vcpkg/commands.build.cpp

```diff
@@ -1664,8 +1670,15 @@ namespace vcpkg
             }
         }
 
-        std::string version = parser.optional_field("Version");
-        if (!version.empty()) build_info.version = std::move(version);
+        auto version_text = parser.optional_field(Fields::VERSION);
+        auto port_version_text = parser.optional_field(Fields::PORT_VERSION);
+        if (!version_text.empty())
+        {
+            auto version =
+                Version::parse(version_text, port_version_text.empty() ? Optional<StringView>{} : port_version_text)
```

Comment:
Maybe `port_version_text.value_or(Optional<StringView>{})`?

Reply:
`port_version_text` is a `std::string` that we are turning into an optional. It doesn't have a `value_or`.